setup.py

```diff
@@ -4,12 +4,15 @@
 import os
 from setuptools import setup, find_packages
 
+import formidable
+
+
 with open(os.path.join(os.path.dirname(__file__), 'README.rst')) as readme:
     README = readme.read()
 
 setup(
     name='django-formidable',
-    version='0.6.0.dev0',
+    version=formidable.version,
```

Comment:
will it work when installing the package on a fresh venv? can you really import your package before installing it?